LayoutTests/TestExpectations

```diff
@@ -5273,6 +5273,7 @@ imported/w3c/web-platform-tests/css/css-images/tiled-conic-gradients.html [ Skip
 
 # Untriaged counter failures
 webkit.org/b/214457 imported/w3c/web-platform-tests/css/css-lists/change-list-descendant-display.html [ ImageOnlyFailure ]
+imported/w3c/web-platform-tests/css/css-lists/change-list-style-position-002.html [ ImageOnlyFailure ]
```

Comment:
Is it significant difference because else it would be regression in this test. Don't know whether it is OK or not. @alanbaradlay - might have some insight.